docs/chapters/baseplatform/replacing-packagegroups.rst

```diff
@@ -0,0 +1,57 @@
+:orphan:
+
+.. _replacing-packagegroups:
+
+Replacing package groups by hand-picked components
+==================================================
+
+By default PELUX include components from QT Auto using the package groups feature.
+This chapter describes how to replace these default QT Auto components with the
+handpicked components.
+
+Following package groups from Qt Auto are included in the PELUX image. These package groups
+add several QT Auto add-ons (includes ``neptune-ui`` among others) and packages that are used 
+when building SDK for host and target machines.
+
+* ``packagegroup-b2qt-automotive-addons``
+* ``nativesdk-packagegroup-b2qt-automotive-qt5-toolchain-host``
+* ``packagegroup-b2qt-automotive-qt5-toolchain-target``
+
+``packagegroup-b2qt-automotive-addons`` includes ``neptune-ui`` component among others which is 
+a reference Qt Auto UI. However, in case a specific distribution do not want to use Qt Auto 
```

Comment:
"reference _to_ "

Reply:
May be "Qt Auto reference UI" makes more sense, as it is a reference UI and not a reference to a UI.

Reply:
Oh I see now. Yes, I agree.